projects/portfolio/portfolio-webpage-1.md

```diff
@@ -73,12 +73,13 @@ You should write clear + informative commit messages for each commit and create
 
 **Create your project**
 
-- [ ] Use bash to create a local portfolio-website directory. Use the command line to make an portfolio.html and an portfolio.css in the portfolio-website directory.
-- [ ] Use the command line to change the names of portfolio.html and portfolio.css to index.html and index.css.
-- [ ] Use the command line to make a gitignore file in your desktop directory and add at least one real unwanted file to the gitignore.
-- [ ] Use the command line to move the gitignore file from your desktop directory to your portfolio-website directory.
-- [ ] Using your command line, initialize a GitHub repo in portfolio-website.
-- [ ] Create a remote repo for the project in your GitHub account.
+- [ ] Use bash to create a local portfolio-website directory. Use the command line to make a portfolio.html file and a portfolio.css file in the portfolio-website directory.
```

Comment:
Curious, Why we have to create the `portfolio.html` and `portfolio.css` first and then rename it, since we ask to create the file under portfolio-website directory? Feels like it will make the student a bit confused.

Reply:
thank you @thisIsTheWei that's a great observation. I think it is practice using Git to create and rename files but let me ask @daaimah123 to confirm the purpose!

Reply:
Yes, that is an intentional step for both using git commands, but then changing file name to a more conventional naming

Reply:
@thisIsTheWei @daaimah123  Thank you Daaimah for explaining the intention behind this exercise. I will leave it as is, since the instructions are clear.